Makefile

```
@@ -184,17 +187,22 @@ eclint/lint:
 	$(ECLINT_FILES) | grep -zv ".bats" | xargs -0 -I {} eclint check {}
 	@ echo "[$@]: Project PASSED eclint test!"
 
-python/%: FIND_PYTHON := find . $(FIND_EXCLUDES) -name '*.py' -type f
-## Lints Python files
-python/lint: | guard/program/black
+python/%: PYTHON_FILES := git ls-files '*.py'
```

I like using `git ls-files ...` here, but let's change it to `?=` so projects can override the list. This will replace the FIND_EXCLUDES override feature that worked before...

```suggestion
python/%: PYTHON_FILES ?= git ls-files '*.py'
```

---

I've been working on the test and the tests need to use git files now, so if someone changes the command used to retrieve the python files, they'd have to change the tests as well.  Unless you're referring to changing the command on a one-off basis that would not involve rerunning make test.